src/main/kotlin/lotto/domain/LottoStore.kt

```diff
@@ -1,18 +1,31 @@
 package lotto.domain
 
-class LottoStore(private val userMoney: UserMoney, private val lottoMaker: LottoMaker = KoreanLottoNumberMaker()) {
-    val lottoCount = userMoney.getLottoCount()
-    val boughtLottos = List(lottoCount) { lottoMaker.makeLottoNumbers() }
+class LottoStore(
+    private val money: Money,
+    manualLottos: List<LottoNumbers>,
+    autoLottos: List<LottoNumbers>
+) {
+    private val _boughtLottos = mutableListOf<LottoNumbers>()
```

Comment:
아래처럼 작성할 수 있을 것 같아요 :) 
```suggestion
    private val _boughtLottos = manualLottos + autoLottos
```